src/gp-model/main.py

```diff
@@ -118,9 +128,16 @@ def train_model(X, Y, airqloud):
         m.likelihood.variance.assign(400)
         set_trainable(m.kernel.kernels[0].variance, False)
         set_trainable(m.likelihood.variance, False)
+    elif airqloud == 'kira':
```

Comment:
the elif and else have the same code, potential code duplication. @lilly-muyama

Reply:
@Baalmart This has been worked on

Reply:
Thanks @lilly-muyama